src/dao/pages/MultiSigDAODashboard/MultiSigDAODashboard.tsx

```diff
@@ -45,7 +45,7 @@ export function MultiSigDAODashboard() {
         errorMessage={errorMessage}
         isSuccess={isSuccess}
       >
-        <Outlet context={{ dao, tokenBalances, members, totalAssetValue }} />
+        <Outlet context={{ dao, tokenBalances, members, totalAssetValue, blockedBalance: 0 }} />
```

Comment:
Add type safety for the `Outlet` context and `useOutletContext` hook so Typescript can point out if a similar issue arises in the future.

Reply:
As per suggestion I added the type for `blockedBalance` and also maintained the undefined condition. I was under impression that undefined was already handled for `AssetList` now its done. let me know if I understand this correct.

Reply:
Looks like the `satisfies` keyword from TS v4.9+ will help with the typing: https://www.youtube.com/watch?v=49gHWuepxxE